### PR TITLE
BUG: When writing a MetaIO file, the first line must be ObjectType

### PR DESCRIPTION
**Pull request by aylward**
This rule was followed when writing all objects, except metaScene.  This commit fixes metaScene.  Additionally, MET_Read/ReadType will look at the first three tag/value pairs in a meta file to determine an object type, so it is more robust, but it will throw warnings regarding unknown fields if the first line isn't ObjectType.

### CODE REVIEW

File: src/metaScene.cxx [hunk 1]
```diff
@@ -168,16 +168,16 @@ MetaScene::Read(const char * _headerName)
     const std::string objectType = MET_ReadType(*m_ReadStream);
     if (!strncmp(objectType.c_str(), "Tube", 4) || ((objectType.empty()) && !strcmp(suf, "tre")))
     {
-      char * subtype = MET_ReadSubType(*m_ReadStream);
-      if (!strncmp(subtype, "Vessel", 6))
+    const std::string subtype = MET_ReadSubType(*m_ReadStream);
+      if (!strncmp(subtype.c_str(), "Vessel", 6))
```
Comment (dzenanz) [inline src/metaScene.cxx:172]: Why not switch this to a regular string comparison? e.g. `if (subtype == "Vessel")`. Perhaps add an additional style commit?

File: src/metaUtils.cxx [hunk 2]
```diff
@@ -193,25 +193,23 @@ MET_ReadSubType(std::istream & _fp)
...
-  // Find the line right after the ObjectType
-  char s[1024];
-  _fp.getline(s, 500);
-  std::string value = s;
-  size_t      position = value.find('=');
-  if (position != std::string::npos)
+  if (mF->defined)
   {
-    value = value.substr(position + 2, value.size() - position);
+    std::string value = reinterpret_cast<char *>(mF->value);
```
Comment (dzenanz) [inline src/metaUtils.cxx:206]: This refactoring is a bit hard to follow.

**Review by dzenanz — APPROVED** [review]
Mostly looks good on a glance.

File: src/metaScene.cxx (hunk 1 above)
Comment (aylward) [reply src/metaScene.cxx:172]: Doh!  Was following old pattern in the file.  I'll update this and other tests.  Thanks!

**Review by dzenanz — APPROVED** [review]
